### PR TITLE
treewide: http -> https

### PR DESCRIPTION
**Pull request by ryantm**
more work on changing to TLS URLs

### CODE REVIEW

**Comment by Infinisil** [comment]
Is this automated? Because I just got something to work that could do this automatically, even for most urls in `fetchurl` and similar

**Comment by ryantm** [comment]
@Infinisil Nope, I'm just manually going through the repology problems using https://github.com/samueldr/repology-autofixer `problems.rb`

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -17523,7 +17523,7 @@ EOF
     doCheck = false;
     meta = {
       description = "A client side encrypted pastebin";
-      homepage = "http://0bin.net/";
+      homepage = https://0bin.nbet/;
```
Comment (xeji) [inline pkgs/top-level/python-packages.nix]: `s/nbet/net/`

Comment (ryantm) [reply pkgs/top-level/python-packages.nix]: Thanks! Fixed.

**Comment by xeji** [comment]
Why does this rebuild 3 vim plugin packages according to ofborg?

**Pull request merged**